discord/commands/context.py

```diff
@@ -134,7 +134,17 @@ def response(self) -> InteractionResponse:
 
     @property
     def respond(self):
-        return self.followup.send if self.response.is_done() else self.interaction.response.send_message
+        if not self.response.is_done():
+            return self.interaction.response.send_message
+        else:
+            raise RuntimeError(f"Interaction was already issued a response. Try using {type(self).__name__}.send_followup() instead.")
+    
```

Comment:
```suggestion

    @property
    async def send_response(self):
        """Callable[..., Union[Interaction, Webhook]]: Sends either a response or a followup response depending if the interaction has been responded to yet or not.
        if not self.response.is_done():
            return self.interaction.response.send_message  # self.response
        else:
            return self.followup.send  # self.send_followup
```

Reply:
This in my view just returns us to the state before this PR, making this PR pointless in the first place. Maybe implementing a `send_or_followup()` func could be something that could be done, but that is outside the scope of this PR.